src/lib/update-file-in-branch.ts

```diff
@@ -131,14 +131,3 @@ async function updateFileInBranch(options) {
     await processRepository(repository, branch, path, patchFunction, message);
   }
 }
-
-/** Callback function that performs required change to the file. The function
- * may apply a patch, or parse and change the file, or do whatever it needs.
- * @callback patchFunction
```

Comment:
This is here because I refer to `patchFunction` above in line 93. I found this is the correct way of describing callbacks in jsdoc. Should it be gone now?